src/marqo/tensor_search/models/settings_object.py

```diff
@@ -128,7 +128,7 @@
                                 NsFields.hnsw_ef_construction: {
                                     "type": "integer",
                                     "minimum": 1,
-                                    "maximum": int(read_env_vars_and_defaults(EnvVars.MARQO_EF_CONSTRUCTION_MAX_VALUE)),
+                                    "maximum": read_env_vars_and_defaults_ints(EnvVars.MARQO_EF_CONSTRUCTION_MAX_VALUE),
```

Comment:
may be the wrong PR to comment this on, but maybe we should rename `MARQO_EF_CONSTRUCTION_MAX_VALUE` to `MARQO_MAX_EF_CONSTRUCTION_VALUE`, because if you observe the other env vars, the word `MAX` always comes first. It may confuse users if we keep EF like that.

Reply:
good thinking, to keep things consistent, but that would technically be a breaking change